docs/src/man/joins.md

```diff
@@ -219,3 +200,41 @@ julia> join(a, b, on = [:City, :Job])
 ```
 
 Additionally, notice that in the last join rows 2 and 3 had the same values on `on` variables in both joined `DataFrame`s. In such a situation `:inner`, `:outer`, `:left` and `:right` joins will produce all combinations of matching rows. In our example rows from 2 to 5 were created as a result. The same behavior can be observed for rows 4 and 5 in both joined `DataFrame`s.
+
+In order to check if that columns passed as the `on` argument define unique keys in each input data frame (according to `isequal`) you can set a `validate` keyword argument to a two element tuple or a pair of `Bool` values, with the first element indicating whether to run check for the first and the second element for the second data frame. Here is an example for the join operation described above:
+
+```jldoctest joins
+julia> join(a, b, on = [(:City, :Location), (:Job, :Work)], validate=(true, true))
+ERROR: ArgumentError: Merge key(s) are not unique in both df1 and df2. First duplicate in df1 at 3. First duplicate in df2 at 3
+```
+
+Finally, using `indicator` keyword argument you can add a column to the resulting data frame giving an information if the given row appeared only in the left, the right or both data frames. Here is an example:
+
+```jldoctest joins
+julia> a = DataFrame(ID = [20, 40], Name = ["John", "Jane"])
```

Comment:
Can't we use the existing data frames here?

Reply:
No, because in the existing data frames all keys match so we would see `"both"` everywhere.